solr_reindex/reindex.sh

```diff
@@ -30,6 +35,11 @@ done
 JOBS_DEFAULT=$(expr \( $(find /sys/devices/system/cpu -maxdepth 1 -type d -regex '.*/cpu[0-9]+$' | wc -l) + 1 \) / 2)
 MEM_BUFF=$(expr $(grep MemTotal /proc/meminfo | awk '{print $2}') / 10)
 
+if [ -z "$LIMIT_DAYS_AGO" ]
```

Comment:
Should this be a `!`? With how it currently is the `LIMIT_ARG` never gets added to the find command.